crates/sui-core/src/authority_active/gossip/mod.rs

```diff
@@ -104,10 +104,10 @@ where
 
             // If we have already used all the good stake, then stop here and
             // wait for some node to become available.
-            let total_stake_used: usize = peer_names
+            let total_stake_used = peer_names
                 .iter()
                 .map(|name| committee.weight(name))
-                .sum::<usize>()
+                .sum::<u64>()
```

Comment:
This is an example place where different entries may end up being from different committees?

Reply:
I think we can make this particular one safe by cloning the committee at the beginning of the loop.